api/HTMLMediaElement.json

```diff
@@ -87,6 +160,27 @@
                   "value_to_set": "true"
                 }
               ]
+            },
+            "chrome": {
```

Comment:
Would you mind adding these entries in sorted order?

Reply:
This is now actually required. Updated this.